src/main.rs

```diff
@@ -324,9 +324,22 @@ fn main() -> ExitCode {
         .contains("nixpkgs=");
 
     if args.install {
-        let _ = Command::new("nix-env")
-            .args(["-f", "<nixpkgs>", "-iA", basename])
-            .exec();
+        let home_dir = PathBuf::from(std::env::var_os("HOME").unwrap_or_default());
+        let profile_dir = home_dir.join(".local/state/nix/profiles/profile");
```

Comment:
The `$NIX_STATE_HOME` and `$XDG_STATE_HOME` vars have to considered too with the fallback being `$HOME/.local/state...`

Or we could just check if `nix profile list` is not empty

Reply:
Uhh, well, I think [the docs](https://nix.dev/manual/nix/2.28/command-ref/files/profiles#user-profile-link) specify that `~/.nix-profile` always points to the current user's profile, no matter what. Isn't it easier if I just check for that link?